backend/src/main/java/com/htecgroup/skynest/service/impl/BucketServiceImpl.java

```diff
@@ -71,6 +68,7 @@ public BucketResponse createBucket(BucketCreateRequest bucketCreateRequest) {
 
   @Override
   public BucketResponse getBucketDetails(UUID uuid) {
+    permissionService.currentUserHasPermissionForBucket(uuid, AccessType.VIEW);
```

Comment:
Is this gohna work if he has a higher access type?

Reply:
I'm pretty sure yes, see the currentUserHasPermissionForBucket method.

Reply:
My bad, didn't see the function accepts minimumAccessType, then I believe this is good

Reply:
It's written by @ivanfhtec, of course it's good.